### PR TITLE
Update broken links

### PR DESCRIPTION
**Pull request by fakela**
This pull request is to fix the broken links in the files accordingly below, the checkbox is files being fixed. The files not listed here have already been fixed or have no links in them

- [x] Getting started docs doc.en.md

- [x] Cron jobs doc.en.md

- [x] Variable doc.en.md

- [x] Troubleshooting

- [x] Web notifications

- [x] Plugin

- [x] clear bit

- [x] connecwise

- [x] fullcontact

- [x] gmail

- [x] bounce management 

- [x] device graularity

- [x] managing pages 

- [x] gmail

- [x] hubspot

- [x] icontact

- [x] mailchimp

- [x] crm 

- [x] Microsoft outlook

- [x] social login

- [x] sugar crm

- [x] twitter

- [x] vtiger

- [x] zapier

- [ ] 

- [ ] 

- [ ]

### CODE REVIEW

File: pages/19.Translations/docs.en.md [hunk 1]
```diff
@@ -60,4 +60,9 @@ A language is downloaded automatically every time the configuration is saved and
 
 The language should be downloaded again with the latest translations. The translations are generated from Transifex once a day.
 
-If you have any questions about translations, join the community in the [Slack #Translations channel](https://www.mautic.org/slack/).
+If you have any questions about translations, join the community in the [Slack #Translations channel][Slack #Translations channel].
+
+
+[Transifex]:<https://www.transifex.com/mautic/mautic/>
+[Transifex Documentation]:<http://docs.transifex.com/tutorials/txeditor/>
+[Slack #Translations channel]:<https://www.mautic.org/slack/>
```
Comment (RCheesley) [inline pages/19.Translations/docs.en.md:68]: Can we change this to a direct link to the channel: https://mautic.slack.com/archives/C02HV79J2

File: pages/19.Translations/docs.en.md [hunk 2]
```diff
@@ -60,4 +60,9 @@ A language is downloaded automatically every time the configuration is saved and
 
 The language should be downloaded again with the latest translations. The translations are generated from Transifex once a day.
 
-If you have any questions about translations, join the community in the [Slack #Translations channel](https://www.mautic.org/slack/).
+If you have any questions about translations, join the community in the [Slack #Translations channel][Slack #Translations channel].
+
+
+[Transifex]:<https://www.transifex.com/mautic/mautic/>
```
Comment (RCheesley) [inline pages/19.Translations/docs.en.md:66]: See earlier note on leaving a space after the :

**Comment by fakela** [comment]
@RCheesley I have made edits, double-checked on my own end, waiting for your feedback and review

File: pages/12.Plugins/05.Connectwise/docs.en.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ This plugin can push/pull a contact to/from Connectwise Manage.
 
 You will need your company id in order to authenticate the Connectwise plugin.
 
-Use the thick client to create a new API member.  Dowload think client from [here](https://university.connectwise.com/university/pageview.aspx?short_name=workstation-installation). 
+Use the thick client to create a new API member.  Dowload think client from [here][connectwise–thick-client].
```
Comment (RCheesley) [inline pages/12.Plugins/05.Connectwise/docs.en.md:26]: Tiny typo!

```suggestion
Use the thick client to create a new API member.  Download think client from [here][connectwise–thick-client]. 
```

File: pages/12.Plugins/12.Microsoft Outlook/docs.en.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ Even if the plugin is compatible to Outlook 2013, this will describe the install
 2. Click on the Outlook plugin button and enter a secret or key to validate the Outlook Add-In
 ![image](secret.png)
 
-3. Run the [Mautic Outlook Add-In Installer](https://m.mautic.com/asset/24:microsoft-outlook-plugin-102) on a Windows machine with Outlook 2016
+3. Run the [Mautic Outlook Add-In Installer][add-in-installer]on a Windows machine with Outlook 2016]
```
Comment (RCheesley) [inline pages/12.Plugins/12.Microsoft Outlook/docs.en.md:35]: Needs a space here I think, and an extra bracket to remove :)

```suggestion
3. Run the [Mautic Outlook Add-In Installer][add-in-installer] on a Windows machine with Outlook 2016
```

File: pages/12.Plugins/14.Salesforce/docs.en.md [hunk 5]
```diff
@@ -262,16 +261,14 @@ Follow [these steps][testing] to test the integration.
 This means the API is not turned on in your Salesforce account. [Read more][salesforce-admin]
 
 
-[Salesforce CRM]: <https://www.salesforce.com/>
+[salesforce-crm]: <https://www.salesforce.com/>
 [salesforce-admin]: <https://help.salesforce.com/apex/HTViewHelpDoc?id=admin_userperms.htm&language=en>
-
+[official-documentation]:<http://feedback.uservoice.com/knowledgebase/articles/235661-get-your-key-and-secret-from-salesforce>
```
Comment (RCheesley) [inline pages/12.Plugins/14.Salesforce/docs.en.md:266]: Needs a space after the : 

```suggestion
[official-documentation]: <http://feedback.uservoice.com/knowledgebase/articles/235661-get-your-key-and-secret-from-salesforce>
```

File: pages/12.Plugins/18.VTiger/docs.en.md [hunk 6]
```diff
@@ -37,10 +37,14 @@ In the Features tab is *Push contacts to this integration* checkbox and it is ch
...
 
 Save the plugin configuration.
 
 ## Test the plugin
 
-Follow [these steps](./../plugins/integration_test.html) to test the integration.
+Follow [these steps][testing] to test the integration.
+
+[vTiger-crm]: <https://www.vtiger.com/>
+[[field-mapping]: </plugins/general-resources/field-mapping>
```
Comment (RCheesley) [inline pages/12.Plugins/18.VTiger/docs.en.md:49]: One too many brackets here:

```suggestion
[field-mapping]: </plugins/general-resources/field-mapping>
```

File: pages/12.Plugins/19.Zapier/docs.en.md [hunk 7]
```diff
@@ -115,4 +115,6 @@ Even this integration is developed as an open source software as is usual with M
 
 You may encounter a 404 error during configuration. If this happens, [clear the cache][clear-cache].
 
-[clear-cache]: </troubleshooting#1-clear-the-cache>
+[cache]: </troubleshooting>
```
Comment (RCheesley) [inline pages/12.Plugins/19.Zapier/docs.en.md:118]: Sorry if I was unclear on this one!

The macro used inline is [clear-cache] and the link was correct, so we need:

```suggestion
[clear-cache]: </troubleshooting#1-clear-the-cache>
```

File: pages/12.Plugins/docs.en.md [hunk 8]
```diff
@@ -19,6 +19,6 @@ If you are on a freshly installed Mautic instance, there is a chance that you do
 
 If you are trying to install a new plugin that did not come with the original Mautic installation files and it does not appear after you click on "Install/Upgrade plugins", [clear your Mautic cache][cache] and try the _**Install/Upgrade plugins**_ button again.
 
-[cache]: <./../tips/troubleshooting.html>
+[cache]: </troubleshooting>
```
Comment (RCheesley) [inline pages/12.Plugins/docs.en.md:22]: Let's drop them at the right location:
```suggestion
[cache]: </troubleshooting#1-clear-the-cache>
```

File: pages/12.Plugins/14.Salesforce/docs.en.md (hunk 5 above)
Comment (fakela) [reply pages/12.Plugins/14.Salesforce/docs.en.md:266]: I didn't see this, thanks for picking this up

**Comment by fakela** [comment]
@RCheesley, Thanks a lot for the extra check and picking out errors. Great work on the reviews, suggestions and feedback

**Comment by RCheesley** [comment]
@fakela it looks like we have some conflicts as the Twilio plugin page has altered the numbering of the folders in the plugins directory - can you take a look? Hopefully a quick fix!

**Comment by fakela** [comment]
> @fakela it looks like we have some conflicts as the Twilio plugin page has altered the numbering of the folders in the plugins directory - can you take a look? Hopefully a quick fix!

fixed

**Pull request merged**